R/read_data.R

```diff
@@ -0,0 +1,163 @@
+#' Read in the dataset of incident case counts
+#'
+#' Each row of the table corresponds to a single facilities' cases for a
+#' reference-date/report-date/disease tuple. We want to aggregate these counts
+#' to the level of geographic aggregate/report-date/reference-date/disease.
+#'
+#' We handle two distinct cases for geographic aggregates:
+#'
+#' 1. A single state: Subset to facilities **in that state only** and aggregate
+#' up to the state level 2. The US overall: Aggregate over all facilities
+#' without any subsetting
+#'
+#' Note that we do _not_ apply exclusions here. The exclusions are applied
```

Comment:
What's an exclusion? Note that all data are included in the aggregate?

Reply:
https://github.com/CDCgov/cfa-epinow2-pipeline/pull/17#issuecomment-2330935021

Reply:
See also #25

Reply:
Yep comment is helpful, thanks